### PR TITLE
Milestone 4: Building  With Backgrounds And Gradients

### PR DESCRIPTION
**Pull request by RaminMammadzada**
We have a cloned Apple's webpage so that it will have the following parts:
- Tricky navigation bar
- Background Images using background-image property
- Major use of Gradient colors
- Thumbnail section
- Header section
- Footer section

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+  background-repeat: no-repeat, no-repeat;
+  background-position: 2px 2px;
+  border-radius: 12px;
+}
+
+input:focus {
+  width: 140px;
+  padding-left: 20px;
+
+  /* margin: 8px 9px; */
```
Comment (elmejdki) [inline css/style.css]: Please remove all commented code from your HTML and CSS files.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            </h2>
+            <h5>
+                <a  href="#">Watch the “Your Verse” film and explore the stories</a>
+            </h5>
+        </header>
+        <div class="row">
+            <div class="col-3 mobile-imag1"></div>
+            <div class="col-3 mobile-imag2"></div>
+            <div class="col-3 mobile-imag3"></div>
+            <div class="col-3 mobile-imag4"></div>
```
Comment (elmejdki) [inline index.html]: please wrap each one of these images inside a link tag, like the original page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    <li><a href="#">Store</a></li>
+                    <li><a href="#">Mac</a></li>
+                    <li><a href="#">iPod</a></li>
+                    <li><a href="#">iPhone</a></li>
+                    <li><a href="#">iPad</a></li>
+                    <li><a href="#">iTunes</a></li>
+                    <li><a href="#">Support</a></li>
+                    <li><input type="search" /></li>
+                </ul>
+            </nav>
```
Comment (elmejdki) [inline index.html]: **[OPTIONAL]** Close the header tag here and open the main tag after it that will hold the main section, and use position absolute for the header tag so you make it overflow the main section. in order to make your code more semantic

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+  transform: scale(0.8);
+}
+
+footer a {
+  text-decoration: none;
+  color: #08c;
+}
+
+.logo a {
+  padding: 22px 30px 1px 28px !important;
```
Comment (elmejdki) [inline css/style.css]: Please remove the !important keyword it's highly not recommended you can always find a way to fix the selector without using !important keyword.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job so far team :clap: :clap:, your clone looks almost the same as the original one :tophat:, but you need to make some minor changes to make it approved.
### status: require changes :recycle: 

#### please make all the required changes bellow before you request another pull request.
_If you have any question or need help don't hesitate to contact me on slack_

File: css/style.css (hunk 1 above)
Comment (umairarshadbutt) [reply css/style.css]: done

File: index.html (hunk 2 above)
Comment (umairarshadbutt) [reply index.html]: done

File: css/style.css (hunk 4 above)
Comment (umairarshadbutt) [reply css/style.css]: done

**Pull request merged**